src/rust/metrics/Cargo.toml

```diff
@@ -15,3 +15,9 @@ common = { path = "../common" }
 macros = { path = "../macros" }
 strum = "0.20.0"
 strum_macros = "0.20.1"
+
+[dependencies.rustcommon-metrics]
```

Comment:
added an import here specifically for the `metrics` crate that now depends on rustcommon-metrics directly instead of through `common`

Reply:
Can we avoid this? The reasoning for having everything pulled in through common is so that we have just one Cargo.toml to control the versioning of the dependency.

Reply:
resolved w/ @brayniac offline that I was trying to illustrate an issue between the interplay of re-exporting in the local crate `common` and the `metric` macro from Rustcommon. I didn't want to actually commit this change to master and we're on the same page now. 

By explicitly setting the `crate` attribute, we avoid the previously mentioned issue at the cost of verbosity